src/ripple/app/ledger/MetaView.h

```diff
@@ -150,41 +238,79 @@ class MetaView : public View
     void
     update (std::shared_ptr<SLE> const& sle) override;
 
-    bool
-    openLedger() const override;
-
     //--------------------------------------------------------------------------
 
-    /** Apply changes to the parent View */
+    /** Apply changes to the base View.
+
+        `to` must contain contents identical to the
+        parent view passed upon construction, else
+        undefined behavior will result.
+
+        After a call to apply(), the only valid operation
+        on the object is a call to the destructor.
+    */
     void
-    apply();
+    apply (BasicView& to,
+        beast::Journal j = {});
+
```

Comment:
Given the comment just above this function, I like Scott's suggestion.  However this:
http://blogs.msdn.com/b/vcblog/archive/2014/06/11/c-11-14-feature-tables-for-visual-studio-14-ctp1.aspx
suggests that VS might not like it until we upgrade to VS-2015.